src/components/formio-wrapper.js

```diff
@@ -676,11 +686,10 @@ export class FormioWrapper {
     this._updateIfCompleted(pageNo, this.wizard.pages);
 
     // Oddness... seems it doesn't set page the first time.
-    this.wizard.setPage(pageNo)
-      .then(() => {
-        this.wizard.setPage(pageNo);
-        this.wizard.redraw();
-      });
+    this.wizard.setPage(pageNo).then(() => {
```

Comment:
I bet what is causing this is that the Wizard is setting the page immediately after this page is set. What may help is that you add this logic inside of a "render" event handler since this is called after the page has been set, rendered, and attached.

https://github.com/formio/formio.js/blob/master/src/Wizard.js#L294